code/game/objects/items/toys.dm

```diff
@@ -503,207 +503,324 @@
 /*
  * Action figures
  */
-
 /obj/item/toy/figure
-	name = "Completely Glitched action figure"
-	desc = "A \"Space Life\" brand... wait, what the hell is this thing? It seems to be requesting the sweet release of death."
-	icon_state = "assistant"
+	name = "Non-Specific Action Figure action figure"
+	desc = "A \"Space Life\" brand... wait, what the hell is this thing?"
 	icon = 'icons/obj/toy.dmi'
+	icon_state = "nuketoy"
+	var/cooldown = 0
+	var/toysay = "What the fuck did you do?"
+
+/obj/item/toy/figure/New()
+	..()
+	desc = "A \"Space Life\" brand [name]"
+
+/obj/item/toy/figure/attack_self(mob/user as mob)
+	if(cooldown < world.time)
+		cooldown = (world.time + 30) //3 second cooldown
+		user.visible_message("<span class='notice'>The [src] says \"[toysay]\".</span>")
+		playsound(user, 'sound/machines/click.ogg', 20, 1)
 
 /obj/item/toy/figure/cmo
 	name = "Chief Medical Officer action figure"
 	desc = "A \"Space Life\" brand Chief Medical Officer action figure."
 	icon_state = "cmo"
+	toysay = "Suit sensors!"
 
 /obj/item/toy/figure/assistant
 	name = "Assistant action figure"
 	desc = "A \"Space Life\" brand Assistant action figure."
 	icon_state = "assistant"
+	toysay = "Grey tide station wide!"
 
 /obj/item/toy/figure/atmos
 	name = "Atmospheric Technician action figure"
 	desc = "A \"Space Life\" brand Atmospheric Technician action figure."
 	icon_state = "atmos"
+	toysay = "Glory to Atmosia!"
 
 /obj/item/toy/figure/bartender
 	name = "Bartender action figure"
 	desc = "A \"Space Life\" brand Bartender action figure."
 	icon_state = "bartender"
+	toysay = "Wheres my monkey?"
 
 /obj/item/toy/figure/borg
 	name = "Drone action figure"
 	desc = "A \"Space Life\" brand Drone action figure."
 	icon_state = "borg"
+	toysay = "I. LIVE. AGAIN."

```

Comment:
maybe bold rather than allcaps

Reply:
actually perhaps use another font for this one?

Reply:
I'd use console but I don't know the 'ms-font' names